osquery/distributed/distributed.cpp

```diff
@@ -78,7 +79,9 @@ Status Distributed::pullUpdates() {
 
 size_t Distributed::getPendingQueryCount() {
   WriteLock lock(distributed_queries_mutex_);
-  return queries_.size();
+  std::vector<std::string> rocks_queries;
```

Comment:
You're not always using RocksDB, so the `rocks_` here is conflating the meaning if this vector.